core/src/main/java/org/mskcc/cbio/portal/scripts/UpdateGenePanel.java

```diff
@@ -185,6 +207,35 @@ public UpdateGenePanel(String[] args) {
      */
     public static void main(String[] args) {
         ConsoleRunnable runner = new UpdateGenePanel(args);
-        runner.runInConsole();
+        Scanner s = new Scanner(System.in);
+        boolean confirmed = false;
+
+        System.err.println("WARNING: This script is ONLY for updating a gene panel, not for adding a new gene panel. To add a new gene panel, run the ImportGenePanel script.");
```

Comment:
System.err.println() displays red text in Eclipse console. 
System.out.println() displays black text.